src/ConnectedMachine/ConnectedMachine.Autorest/README.md

```diff
@@ -265,6 +265,12 @@ directive:
       parameter-name: AgentUpgradeEnableAutomaticUpgrade
     set:
       parameter-name: AgentUpgradeEnableAutomatic
+    
+  # Rename Tag to Tags
```

Comment:
We prefer to use a singular name for parameter. Further, I don't the doc update for `Tag`